src/views/Stake/StakeV2.js

```diff
@@ -729,6 +733,23 @@ export default function StakeV2({
     setVesterWithdrawAddress(mlpVesterAddress);
   };
 
+  const claimLPAirdrop = () => {
+    // call withdraw on 0x6CFfEC90f2fb63e2b0c03197e75FE919023E727a using proof, correct amount, and round = 0
+    const contract = new ethers.Contract(
+      "0x6CFfEC90f2fb63e2b0c03197e75FE919023E727a",
+      FeeDistributor.abi,
+      library.getSigner()
+    );
+    callContract(chainId, contract, "withdraw", [airdropRewardProofData, airdropRewardAmountBN, AIRDROP_ROUND], {
+      sentMsg: "Airdrop claim submitted.",
+      failMsg: "Airdrop Claim failed.",
+      successMsg: "Claimed!",
+      setPendingTxns,
+    }).finally(() => {
+      setIsWithdrawing(false);
```

Comment:
Is this necessary as `isWithdrawing` isn't referenced anywhere else, and is always false is it is never updated

If you intended for this to have future functionality then did you want to add `setIsWithdrawing(true)` to the top of the function?

Reply:
Ah good catch. Not intentional - I just had this there from another block that used it. I'll remove it.